sklearn/tests/test_multiclass.py

```diff
@@ -855,7 +855,6 @@ def test_pairwise_n_features_in():
 
     # Fitting the same models with a precomputed kernel:
     K = X @ X.T
```

Comment:
the comment was more about moving this after ovo-notprecomputed

Reply:
Indeed :)